docs/02-app/02-api-reference/05-next-config-js/redirects.mdx

```diff
@@ -123,6 +123,27 @@ module.exports = {
 }
 ```
 
+Certain cases may arise when you might have to redirect all pages to a specific page, regex path matching paired with wildcard can help with that: 
```

Comment:
This description needs some details about what is matching and what is not.

Theres a similar question here: https://stackoverflow.com/q/62264995/266535

Reply:
@styfle So, I was creating an application for [SRMKZILLA](https://srmkzilla.net) where there is only one page recruitment.srmkzilla.net and there are no other pages. So whenever someone goes to something like recruitment.srmkzilla.net/hey or anything else, I want that to be redirected to the index page, and not show a custom 404 page. 

I could do this by redirecting it inside the custom 404 page but that seems like an extra step because redirects are supported in next config. Would love to hear your thoughts!

Reply:
Also, I have added more context.

Reply:
This feels like a recipe for infinite redirects. I don’t think this is the right place to document it. 

cc @leerob @delbaoliveira